apps/web/src/pages/get-started/components/OpenWorkflowButton.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from '@emotion/styled';
+
+import { colors } from '@novu/design-system';
+import { INotificationTemplate } from '@novu/shared';
+
+import { ROUTES } from '../../../constants/routes.enum';
+import { useCreateTemplateFromBlueprint } from '../../../api/hooks';
+import { errorMessage } from '../../../utils/notifications';
+import { getBlueprintTemplateById, getTemplateById } from '../../../api/notification-templates';
+import { getWorkflowBlueprintDetails } from '../../../utils';
+import { TemplateCreationSourceEnum } from '../../templates/shared';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+import { OnboardingWorkflowRouteEnum } from '../consts/types';
+
+export function OpenWorkflowButton({
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** This component and file are quite overloaded with functionality. I'd suggest we refactor it to improve readability and testability‏